1.infrastructure/README.md

```diff
@@ -155,4 +158,82 @@ eksctl create cluster -f <EXAMPLE_MANIFEST>.yaml
 Note: If you make changes to your nodegroup after this step, you can just run
 ```bash
 eksctl create nodegroup -f <EXAMPLE_MANIFEST>.yaml  # (or any other config file, as required)
-```
\ No newline at end of file
+```
+
+## Capacity Blocks
```

Comment:
Is this section a must have if users are using the `1.infrastructure` to setup a cluster? Or a good to have? Or there are some conditions to use the CB?